tests/phpunit/tests/test-wpml-config.php

```diff
@@ -107,52 +107,93 @@ protected function translate_options( $slug ) {
 	}
 
 	public function test_cf() {
-		wp_set_current_user( 1 ); // To pass current_user_can_synchronize() test
+		wp_set_current_user( 1 ); // To pass current_user_can_synchronize() test.
+		$json_en         = '{"to_translate":"Value 1","not_to_translate":"Value other"}';
+		$json_fr         = '{"to_translate":"Valeur 1","not_to_translate":"Value other"}';
+		$json_fr_unknown = '{"to_translate":"Valeur inconnue","not_to_translate":"Value other"}';
 
 		$pll_admin = new PLL_Admin( $this->links_model );
 		PLL_WPML_Config::instance()->init();
 
 		$en = $from = self::factory()->post->create();
 		self::$model->post->set_language( $from, 'en' );
-		add_post_meta( $from, 'quantity', 1 ); // copy
-		add_post_meta( $from, 'custom-title', 'title' ); // translate
-		add_post_meta( $from, 'bg-color', '#23282d' ); // copy-once
-		add_post_meta( $from, 'date-added', 2007 ); // ignore
+		add_post_meta( $from, 'quantity', 1 ); // `copy`
+		add_post_meta( $from, 'custom-title', 'title' ); // `translate`
+		add_post_meta( $from, 'bg-color', '#23282d' ); // `copy-once`
+		add_post_meta( $from, 'date-added', 2007 ); // `ignore`
+		add_post_meta( $from, 'a_json_meta', $json_en ); // `translate` + encoding.
 
 		$fr = $to = self::factory()->post->create();
 		self::$model->post->set_language( $to, 'fr' );
 		self::$model->post->save_translations( $en, compact( 'en', 'fr' ) );
 
-		// copy
+		// Enable translation for the JSON meta.
+		add_filter(
```

Comment:
I'm not sure to understand the purpose of this filter in the test.

Reply:
It's only to make sure that the whole process is working. But I think this test has little value for this PR afterall 🤔

Reply:
Fixed in [f1796c7](https://github.com/polylang/polylang/pull/1402/commits/f1796c7eb2a973e28afa2a73b213e6119e01b396).